gallery/src/styles/NxList/NxListBulletedExample.html

```diff
@@ -6,44 +6,42 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list nx-list--bulleted">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul>
-    <li class="nx-list__item">
-      item 1
-    </li>
-    <li class="nx-list__item">
-      <span>Item 2</span>
-      <ul class="nx-list nx-list--bulleted">
-        <li class="nx-list__item">
-          item 1
-        </li>
-        <li class="nx-list__item">
-          Item 2
-        </li>
-        <li class="nx-list__item">
-          <span>item 3</span>
-          <ul class="nx-list nx-list--bulleted">
-            <li class="nx-list__item">
-              item 1
-            </li>
-            <li class="nx-list__item">
-              Item 2
-            </li>
-            <li class="nx-list__item">
-              item 3
-            </li>
-          </ul>
-        </li>
-        <li class="nx-list__item">
-          Item 4
-        </li>
-      </ul>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-    </li>
-  </ul>
-</div>
+<h3 class="nx-h3">
```

Comment:
Is there any value in including this header in the example? It's essentially an entirely separate element

Reply:
> Is there any value in including this header in the example? It's essentially an entirely separate element

Actually, I see all the examples are like this, so I guess I can ask the same question about all of them

Reply:
> > Is there any value in including this header in the example? It's essentially an entirely separate element
> 
> Actually, I see all the examples are like this, so I guess I can ask the same question about all of them

I thought about that, I'd like to keep some of them just to show how you apply a title to a list, but they certainly aren't all necessary

Reply:
> > > Is there any value in including this header in the example? It's essentially an entirely separate element
> > 
> > 
> > Actually, I see all the examples are like this, so I guess I can ask the same question about all of them
> 
> I thought about that, I'd like to keep some of them just to show how you apply a title to a list, but they certainly aren't all necessary

I removed a bunch here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/148e89c773adebead4c0c08ccdb326c8b4dbfed8